lib/compat/wordpress-5.8.php

```diff
@@ -19,7 +19,7 @@
 	 *
 	 * @return array Returns the constructed WP_Query arguments.
 	 */
-	function construct_wp_query_args( $block, $page ) {
+	function build_query_vars_from_query_block( $block, $page ) {
```

Comment:
@ntsekouras, should we update the second param to accept an array of settings or `$page` is all that is necessary here?

Reply:
`$page` is enough and I can't think of any other settings for now.